scoreboard/management/commands/wipe_db.py

```diff
@@ -1,30 +1,92 @@
 # Wipe everything in the database without changing the schema.
-from django.core.management.base import BaseCommand, CommandError
+from django.core.management.base import BaseCommand
 from django.db import transaction
 from scoreboard.models import *
 
+
+@transaction.atomic
+def wipe_leaderboard_fields(entity):
+    entity.longest_streak = 0
+    entity.unique_deaths = 0
+    entity.unique_ascs = 0
+    entity.unique_achievements = 0
+    entity.games_over_1000_turns = 0
+    entity.games_scummed = 0
+    entity.total_games = 0
+    entity.wins = 0
+    entity.trophies.clear()
+    entity.save()
+
+
+@transaction.atomic
+def clear_player_and_clan_fields():
+    for player in Player.objects.all():
+        wipe_leaderboard_fields(player)
+    for clan in Clan.objects.all():
+        wipe_leaderboard_fields(clan)
+
+
+@transaction.atomic
+def wipe_games():
+    Game.objects.all().delete()
+    clear_player_and_clan_fields()
```

Comment:
Wiping games also means resetting the xlog fpos for all Sources. (Any of these wipes that isn't straight up deleting all Sources should do that.)

Reply:
oh yeah true